packages/compiler/core/scanner.ts

```diff
@@ -948,10 +948,9 @@ function skipMultiLineComment(input: string, position: number): number {
       input.charCodeAt(position) === CharCode.Asterisk &&
       input.charCodeAt(position + 1) === CharCode.Slash
     ) {
-      position += 2;
-      break;
+      return position + 2;
     }
   }
 
-  return position;
+  return input.length + 1;
```

Comment:
I think this should be input.length.

Reply:
Hmm, I see then we can't tell if it's unterminated or not. Hmm, we probably should signal that another way then setting position beyond the max value, but not sure the best refactoring.

Reply:
with that it result in it returning the same value if it reached EOF or if the comment ended at the end of the file